2023-08-29 09:56:19,813 - ERROR - data did not save from DB
(psycopg2.errors.UndefinedFunction) ОШИБКА:  оператор не существует: character varying = integer
LINE 3: WHERE users.user_id = 828093617
                            ^
HINT:  Оператор с данными именем и типами аргументов не найден. Возможно, вам следует добавить явные приведения типов.

[SQL: SELECT users.user_id 
FROM users 
WHERE users.user_id = %(user_id_1)s]
[parameters: {'user_id_1': 828093617}]
(Background on this error at: https://sqlalche.me/e/20/f405)
2023-08-29 17:30:50,464 - ERROR - data did not save from DB
(psycopg2.errors.UniqueViolation) ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "users_pkey"
DETAIL:  Ключ "(user_id)=(828093617)" уже существует.

[SQL: INSERT INTO users (user_id, username, city, role) VALUES (%(user_id)s, %(username)s, %(city)s, %(role)s) RETURNING users.date_registration]
[parameters: {'user_id': '828093617', 'username': 'Algernon_LeX', 'city': None, 'role': None}]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2023-09-11 11:35:39,901 - ERROR - data did not save to DB
(sqlalchemy.dialects.postgresql.asyncpg.IntegrityError) <class 'asyncpg.exceptions.UniqueViolationError'>: повторяющееся значение ключа нарушает ограничение уникальности "users_pkey"
DETAIL:  Ключ "(user_id)=(110000011)" уже существует.
[SQL: INSERT INTO users (user_id, username, city, role) VALUES ($1::VARCHAR, $2::VARCHAR, $3::VARCHAR, $4::VARCHAR) RETURNING users.date_registration]
[parameters: ('110000011', 'alex', None, None)]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2023-09-12 18:33:31,104 - ERROR - connect from DB error
__enter__
2023-09-12 21:28:55,885 - ERROR - connect from DB error
__enter__
2023-09-12 21:30:40,754 - ERROR - connect from DB error
__enter__
2023-09-12 21:31:29,336 - ERROR - connect from DB error
__enter__
2023-09-12 21:33:02,542 - ERROR - connect from DB error
__enter__
2023-09-12 21:36:08,604 - ERROR - connect from DB error
__enter__
2023-09-12 21:38:20,989 - ERROR - connect from DB error
__enter__
2023-09-12 21:40:46,868 - ERROR - connect from DB error
__enter__
2023-09-12 21:41:35,570 - ERROR - connect from DB error
__enter__
